Main.js

```diff
@@ -165,34 +167,45 @@ document.addEventListener('DOMContentLoaded', () => {
 
     document.getElementById("set_service_name").addEventListener("click", () => {
 
-        service_name = prompt("Enter the name of the service you wish to track incidents for:")
+        const promptUserIncidentName = new Modal.Modal('Enter the name of the service you wish to track incidents for.', 'Set Service Name', true)
 
-        storage.set("service_name", service_name)
+        promptUserIncidentName.prompt((input) => {
+
+            storage.set("service_name", input)
+
+            document.querySelectorAll('span.service-name').forEach(el => { el.innerHTML = input })
 
-        document.querySelectorAll('span.service-name').forEach(el => {
-            el.innerHTML = service_name
         })
 
     })
 
+    
+    // Reset protocol that creates bulma modals that confirm user wishes to clear local data
+    document.getElementById('reset_everything').addEventListener("click", () => {
 
-    document.getElementById("reset_everything").addEventListener("click", () => {
+        const confirmReset_1 = new Modal.Modal('Do you wish to reset local storage?')
+        const confirmReset_2 = new Modal.Modal('Resetting will delete all data. Are you sure?')
+        const confirmReset_3 = new Modal.Modal("Please type 'YES' to proceed.", 'Danger', true)
```

Comment:
I actually didn't know that function (or in this case constructor) parameters could just not be included as desired, and that as long as the function didn't use them (or in this case set defaults) that the function would still run just fine. I learned something new from your code. Thank you!

Reply:
So long as I learn something new everyday, I am content.